src/makeRobotAccountant.js

```diff
@@ -6,7 +6,28 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let count = 1;
+
+  const addNum = x => {
+    if (x === -Infinity) {
```

Comment:
Checking for x === -Infinity and returning Infinity is not necessary for the task requirements. You can safely remove lines 8-10. But you can leave it at your discretion